awx/main/models/unified_jobs.py

```diff
@@ -1407,7 +1411,14 @@ def awx_meta_vars(self):
         return r
 
     def get_queue_name(self):
+        if self.is_containerized is True:
+            return settings.CELERY_DEFAULT_QUEUE
```

Comment:
Can't quite wrap my head around what this should be but something feels wrong. Why shouldn't `self.container_controller` be returned in this case?

Reply:
Oh sorry, container_controller isn't a model property .. just a temp var. Still, something doesn't feel right here. I think this conditional is not needed and the below return captures all the logic needed.